ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -18,6 +18,18 @@ export const getServerSideProps: GetServerSideProps = async context => {
   );
 
   const { renderer_env: rendererEnv } = context.query as PageDataParams;
+
+  const defaultMetadata = {
+    title: 'BBC World Service – BBC News in 43 Languages Worldwide',
```

Comment:
Will we need to update this every time we add a new language? Is there a way in which we could determine the number of languages programatically?

```suggestion
    title: 'BBC World Service – BBC News in 43 Languages Worldwide',
```

Reply:
I think we don't need default metadata. This information should always come from TIPO.